app/services/course/assessment/submission/update_service.rb

```diff
@@ -1,14 +1,13 @@
 # frozen_string_literal: true
 class Course::Assessment::Submission::UpdateService < SimpleDelegator
   def update
-    if params[:answer_id]
+    if auto_grade?
       submit_answer
```

Comment:
I think the submit of the single answer should be moved to the answers controller. Not sure how the frontend construct the JSON for a single answer, would that be easier for frontend ? @jeremyyap